src/os_posix/os_union_fs.c

```diff
@@ -875,14 +866,15 @@ __union_fs_open_in_source(
 
 #include <unistd.h>
 /*
- * __union_build_extents_from_dest_file_lseek --
- *     When opening a file from destination create its existing extent list from the file system
+ * __union_build_holes_from_dest_file_lseek --
+ *     When opening a file from destination create its existing hole list from the file system
  *     information. Any holes in the extent list are data that hasn't been copied from source yet.
  */
-static void
-__union_build_extents_from_dest_file_lseek(
+static int
+__union_build_holes_from_dest_file_lseek(
```

Comment:
Can we rename this function to: `__union_find_holes_in_dest_file`? We no longer support `fiemap` so it seems redundant. We may also want this to become `__union_fh_find_holes_in_dest_file` but that will come later once we break this c file into many c files.

Reply:
Renamed to `__union_fh_find_holes_in_dest_file` in https://github.com/wiredtiger/wiredtiger/pull/11268/commits/4d4cf9f48bd9fc9e5e3755658d8527ca4865ced7.